components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +267,35 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
```

Comment:
It will likely get re-run.  That shouldn't be a problem though, as most (all?) of the SQL I've seen in our migrations is idempotent.  If it's not, it should be.